forc-test/src/lib.rs

```diff
@@ -189,12 +190,15 @@ impl<'a> PackageTests {
 
     /// Run all tests for this package and collect their results.
     pub(crate) fn run_tests(&self) -> anyhow::Result<TestedPackage> {
+        // TODO: Remove this once https://github.com/FuelLabs/sway/issues/3947 is solved.
```

Comment:
Can you open an issue linking to #3947 as a reminder to remove these changes after that lands?

Reply:
Good idea!